[terra-slide-panel] Assign unique ID to hidden text (#1657)
cerner/terra-framework: packages/terra-framework-docs/src/terra-dev-site/doc/slide-panel/SlidePanel.1.doc.mdx

## Slide Panel Props Table 
<SlidePanelPropsTable />

## Testing

> The Slide Panel uses `uuid` which changes the component's description id dynamically. To mock it in jest test teams can use `jest.mock`.

Reviewer: nit, even though it's copied can we update:
```suggestion
The Slide Panel uses `uuid` which changes the component's description id dynamically. To mock it in Jest tests teams can use `jest.mock`.
```

Author: Done: https://github.com/cerner/terra-framework/pull/1657/commits/a8531250eb7e1c1871507c8eb6241e14370af313

> The Slide Panel uses `uuid` which changes the component's description id dynamically. To mock it in Jest tests teams can use `jest.mock`.

Reviewer 2: ```suggestion
Terra Slide Panel uses `uuid` to dynamically generate the component's description id. It can be mocked in Jest though `jest.mock`. An example is given below.
```

Author: Done: https://github.com/cerner/terra-framework/pull/1657/commits/c9ad6244180cd82a625d1c72c1f688c034c6c9c6

> If teams are using `shallow` for the tests then the mock is not required. But if using `mount` then the teams must use the mock as given below.

Reviewer 2: ```suggestion
If teams are using `shallow` for the tests then the mock is not required. But if the tests use `mount`, then the teams must use the mock as given below.
```

Author: Done: https://github.com/cerner/terra-framework/pull/1657/commits/c9ad6244180cd82a625d1c72c1f688c034c6c9c6
